docker/django.dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM alpine:3.9
```

Comment:
Could we bump up the version up to the actual `3.15`?

Reply:
Sure, let me do that!